tests/unit/service/test_feature_list.py

```diff
@@ -155,26 +155,26 @@ async def test_feature_list__contains_relationships_info(
         item_table,
     )
 
-    # check these relationships are included in the feature list
+    # check only descendant relationships are included
     feature_list = await feature_list_service.create_document(
         data=FeatureListServiceCreate(name="my_feature_list", feature_ids=[feature.id])
     )
     relationships_info = feature_list.relationships_info
-    assert len(relationships_info) == 5
-    expected_entities = set(ancestor_ids + descendant_ids + sibling_ids + [target_entity_id])
+    assert len(relationships_info) == 2
+    expected_entities = set(descendant_ids + [target_entity_id])
```

Comment:
Note that new relationship extraction logic keeps bare minimum entity relationship information.